checkstyle/spotbugs-exclude.xml

```diff
@@ -44,6 +44,10 @@
     <Match> <!-- does not work from JDK11 onwards https://github.com/spotbugs/spotbugs/issues/756 -->
         <Bug pattern="RCN_REDUNDANT_NULLCHECK_OF_NONNULL_VALUE" />
     </Match>
+    <Match> <!-- fromBytes was not reporting this warning in earlier version of checkstyle -->
+        <Bug pattern="NP_NULL_ON_SOME_PATH_FROM_RETURN_VALUE" />
+        <Method name="fromBytes" />
+    </Match>
```

Comment:
fromBytes method is returning some value from some map and that map we are already building on the top of the class you can see [here](https://github.com/pravega/schema-registry/blob/f1735b250ba5df260837100bc343a180bdccd44e/server/src/main/java/io/pravega/schemaregistry/storage/impl/group/records/TableRecords.java#L42)

now there is no way that it could get NULL from the map

but The bugs still reported by newer version of the checkstyle. 

and becuases of that we are suppressing this.